src/main/java/elise/Task.java

```diff
@@ -0,0 +1,66 @@
+package elise;
+
+/**
+ * Represents a task with modifiable status.
+ */
+public abstract class Task {
```

Comment:
nice! I like how this is abstract to ensure we only instantiate concrete implementations like `ToDo`, `Deadline` etc.